docs/Basic-Guide.md

```diff
@@ -27,21 +25,36 @@ for more information.
        Equivalent or .NET 4.x Equivalent)**
 6. Go to **File** > **Save Project**
 
+## Setting up the Inference Engine
+
+In order to run pre-trained models of agents behaviors, you will need to set-up the 
+[Inference Engine](Inference-Engine.md). The Inference Engine is a general API to
+run Neural Network models in Unity that leverages existing inference libraries such 
```

Comment:
"Neural Network models" --> "neural network models (the kind generated by ML-Agents)"